src/realm/object-store/sync/impl/sync_metadata.cpp

```diff
@@ -214,7 +216,7 @@ void migrate_to_v7(std::shared_ptr<Realm> old_realm, std::shared_ptr<Realm> real
 SyncMetadataManager::SyncMetadataManager(std::string path, bool should_encrypt,
                                          util::Optional<std::vector<char>> encryption_key)
 {
-    constexpr uint64_t SCHEMA_VERSION = 7;
+    constexpr uint64_t SCHEMA_VERSION = 8;
```

Comment:
please add a test for the migration, see "sync metadata: can open old metadata realms"